src/app/pages/send/send-crypto-asset-form/form/btc/use-generate-bitcoin-raw-tx.ts

```diff
@@ -0,0 +1,50 @@
+import { useCallback } from 'react';
+
+import * as bitcoin from 'bitcoinjs-lib';
+import { ECPairFactory } from 'ecpair';
+import * as ecc from 'tiny-secp256k1';
+
+import { BitcoinSendFormValues } from '@shared/models/form.model';
+
+import { btcToSat } from '@app/common/money/unit-conversion';
+import { useGetUtxosByAddressQuery } from '@app/query/bitcoin/address/utxos-by-address.query';
+import {
+  useBitcoinPublicKey,
+  useCurrentBtcAccountAddressIndexZero,
+} from '@app/store/accounts/blockchain/bitcoin/bitcoin-account.hooks';
+
+const ECPair = ECPairFactory(ecc);
+
+const validator = (pubkey: Buffer, msghash: Buffer, signature: Buffer): boolean =>
+  ECPair.fromPublicKey(pubkey).verify(msghash, signature);
+
+// Following examples here: https://github.com/bitcoinjs/bitcoinjs-lib
+export function useGenerateBitcoinRawTx() {
```

Comment:
Sorry I didn't share this yesterday, but here's the commit from my previous test https://github.com/hirosystems/stacks-wallet-web/commit/5af0e126f8171a7e9a641b69d7db8e177ed6bed2